_includes/header.html

```diff
@@ -5,7 +5,9 @@
     <div class="container">
       {% if page.title != "about" %}
         {% assign name = site.title | split: ' ' %}
-        <a class="navbar-brand title font-weight-lighter" href="{{ site.baseurl | prepend: site.url }}/"><span class="font-weight-bold">{{ name | first }}</span> {{ name | last }}</a>
+        {% assign firstname =  name | first %}
```

Comment:
Could we explicitly have 3 variables, `firstname`, `midlenames`, and `lastname`? If the name has multiple parts, then `firstname` and `lastname` will stand for the first and last words in the name and the `middlenames` will be all the intermediate words.